### PR TITLE
Removing the BLS signature from the view-change

### PR DESCRIPTION
**Pull request by ineiti**
As @LefKok validated #2264, this PR removes the BLSCoSi round for the view-change. This makes it faster and simpler.

One more idea would be to avoid the view-change requests completely. But that will break even more stuff. So let's start small...

### CODE REVIEW

File: byzcoin/viewchange.go [hunk 1]
```diff
@@ -169,38 +170,40 @@ func (s *Service) sendNewView(proof []viewchange.InitReq) {
...
+	sb := s.db().GetByID(proof[0].View.ID)
+	req, err := viewchange.NewNewViewReq(sb.Roster, proof)
 	if err != nil {
-		log.Error("Couldn't encode request:", err)
-		return
-	}
-	if !s.verifyViewChange(req.Hash(), pl) {
-		log.Error("Will not ask for view change I cannot verify")
-		return
+		log.Errorf("couldn't create request: %v", err)
```
Comment (tharvik) [inline byzcoin/viewchange.go:177]: Not bailing early?

File: byzcoin/viewchange/viewchange.go [hunk 2]
```diff
@@ -125,7 +127,9 @@ func (c *Controller) Stop() {
 }
 
 // Start begins a blocking process that processes incoming view-requests.
-func (c *Controller) Start(myID network.ServerIdentityID, genesis skipchain.SkipBlockID, initialDuration time.Duration, f int) {
```
Comment (tharvik) [inline byzcoin/viewchange/viewchange.go:128]: Same changes already merged in https://github.com/dedis/cothority/pull/2268 (and some more below)

Comment (ineiti) [reply byzcoin/viewchange/viewchange.go:128]: Yes, true. I re-based it on #2268... I'll rebase it on master.

File: byzcoin/viewchange.go (hunk 1 above)
Comment (ineiti) [reply byzcoin/viewchange.go:177]: yes, correct. Converted the method to return an error, and handle them correctly.

**Comment by ineiti** [comment]
After presenting my idea, I put it back to `WIP`, as it's possible to make it even simpler...

**Comment by ineiti** [comment]
This time I'm sure I will not go the whole way - removing all the `viewchange` and `heartbeat` seems too much for the moment.
This PR will already make it a little bit simpler. Even simpler is for later...

**Pull request merged**